e2e/framework/framework.go

```diff
@@ -0,0 +1,210 @@
+package framework
+
+import (
+	"flag"
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+var fEnv = flag.String("nomad.env", "", "name of the environment executing against")
```

Comment:
Given that this framework is inside nomad already, do we really need the "nomad." prefix for all these flags?